lib/logstash/event.rb

```diff
@@ -16,6 +16,13 @@
 class LogStash::ShutdownEvent; end
 class LogStash::FlushEvent; end
 
+module LogStash
```

Comment:
Hum..around 10 plugins use it https://github.com/search?p=1&q=user%3Alogstash-plugins+LogStash%3A%3ASHUTDOWN&ref=searchresults&type=Code&utf8=%E2%9C%93 (I couldn't find a more accurate search for this)

Alright, lets postpone this to > 1.5